02--Fellowship-of-the-String/README.md

```diff
@@ -0,0 +1,272 @@
+# Fellowship of the String
+
+## Rituals (~1h 15m)
+
+* **Standup Meeting** (~15m)
+* **Three Little Things** (~30m)
+* **Homework Review** (~30m)
+
+## Agenda
+
+### Review
+
+* What _primitive types_ exist in JavaScript?
+* What is an Expression?
+  * What is the simplest Expression?
+  * What is an Operator? What does an Operator require?
+  * What is the result of an Operation? Expression!
+* What is a Statement?
+  * What Operator do we use to denote a Statement?
+  * What other Operators can separate Statements?
+  * Can a Statement contain more than one Expression?
+* What other kinds of Statements are there?
+
+### The Wonderful World of JavaScript: Redux
+
+* What is the Rule of Sequential Operation?
+* Are there exceptions to that rule?
+
+### Reading, Writing, and Running JavaScript
+* How can I _read_ JavaScript files?
+  * in the Finder with QuickLook...
+  * from the command line with `less <filename>`
+  * in a code editor like Atom!
+* How do I _write_ JavaScript files?
+  * in the Finder... (boo)
+  * from the command like with `touch <filename>`
+  * back to that Atom...!
+* How do I _run_ JavaScript?
+  * in the browser?
+  * from the command line with `node <filename>`
+  * in the browser?
+* What if I don't have a file yet? Use the `node` REPL!
+
+### All about Functions
+
+* What is a Function?
+  * How is a Function defined? How is a defined Function invoked?
+  * What is the input? How does a function accept input?
+  * What is the output? How does a function provide output?
+  * What are side-effects? How does a function create side-effects?
+* Do I always have to write my own Functions?
+  * What functions are built-in to JavaScript?
+  * Is there any other way to define or invoke a Function?
+  * What is an anonymous function? What is it used for?
+    * examples with `setTimeout`, `setInterval`
+* How does this relate to Control Flow and the Rule of Sequential Operation?
+
+### A little more about Control Flow
+
+* How do I alter the Control Flow of a program?
+  * How do I _draw_ a decision tree? How do I _write_ one?
+  * What if I have more than one decision to make?
+  * How do I draw that?
+* What if I need to make the same decision again?
+
+### Coding Kata: String Calculator
```

Comment:
Some bullet points for this section:

* What's better than `console.assert`? `chai`...!
* How do I use a library like `chai` in my programs?
* What's better than ugly `AssertionError` messages? `mocha`...!
* How do I use `mocha` in my programs?
  * `npm install mocha`
  * `node_modules/.bin/mocha <filename>`
  * Man, that's ugly, huh?
* Let's keep track of that...
  * `npm init` to create a `package.json` file...
  * set the `test` command to `mocha`
  * now we can run `npm test <filename>` instead!